tests/roles/heat_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,56 @@
+- name: Patch osp-secret for HeatAuthEncryptionKey
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch secret osp-secret --type='json' -p='[{"op" : "replace" ,"path" : "/data/HeatAuthEncryptionKey" ,"value" : "UTYwSGo4UHFickROdTJkRENieUlRRTJkaWJwUVVQZzIK"}]'
+
+- name: deploy podified heat
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch openstackcontrolplane openstack --type=merge --patch '
+    spec:
+      heat:
+        enabled: true
+        apiOverride:
+          route: {}
+        template:
+          customServiceConfig: |
+            [DEFAULT]
+            debug = true
+            [keystone_authtoken]
+            insecure = true
```

Comment:
Looks like something might have changed recently and Heat now fails to connect to Keystone because of the self-signed cert. I'll need to find the CA and have the Heat containers trust the CA. Adding this here to see if the CI jobs will pass in the meantime, or find any additional issues that might exist.